File: ansi.java
public final class Ansi {
    public static String RESET = "\\e[0m";
    public static String BLACK = "\\e[0;30m";
    public static String RED = "\\e[0;31m";
    public static String GREEN = "\\e[0;32m";
    public static String YELLOW = "\\e[0;33m";
    public static String BLUE = "\\e[0;34m";
    public static String PURPLE = "\\e[0;35m";
    public static String CYAN = "\\e[0;36m";
    public static String WHITE = "\\e[0;37m";
    public static String BLACK_BOLD = "\\e[1;30m";
    public static String RED_BOLD = "\\e[1;31m";
    public static String GREEN_BOLD = "\\e[1;32m";
    public static String YELLOW_BOLD = "\\e[1;33m";
    public static String BLUE_BOLD = "\\e[1;34m";
    public static String PURPLE_BOLD = "\\e[1;35m";
    public static String CYAN_BOLD = "\\e[1;36m";
    public static String WHITE_BOLD = "\\e[1;37m";
    public static String BLACK_UNDERLINE = "\\e[4;30m";
    public static String RED_UNDERLINE = "\\e[4;31m";
    public static String GREEN_UNDERLINE = "\\e[4;32m";
    public static String YELLOW_UNDERLINE = "\\e[4;33m";
    public static String BLUE_UNDERLINE = "\\e[4;34m";
    public static String PURPLE_UNDERLINE = "\\e[4;35m";
    public static String CYAN_UNDERLINE = "\\e[4;36m";
    public static String WHITE_UNDERLINE = "\\e[4;37m";
    public static String BLACK_BACKGROUND = "\\e[40m";
    public static String RED_BACKGROUND = "\\e[41m";
    public static String GREEN_BACKGROUND = "\\e[42m";
    public static String YELLOW_BACKGROUND = "\\e[43m";
    public static String BLUE_BACKGROUND = "\\e[44m";
    public static String PURPLE_BACKGROUND = "\\e[45m";
    public static String CYAN_BACKGROUND = "\\e[46m";
    public static String WHITE_BACKGROUND = "\\e[47m";
    public static String BLACK_BRIGHT = "\\e[0;90m";
    public static String RED_BRIGHT = "\\e[0;91m";
    public static String GREEN_BRIGHT = "\\e[0;92m";
    public static String YELLOW_BRIGHT = "\\e[0;93m";
    public static String BLUE_BRIGHT = "\\e[0;94m";
    public static String PURPLE_BRIGHT = "\\e[0;95m";
    public static String CYAN_BRIGHT = "\\e[0;96m";
    public static String WHITE_BRIGHT = "\\e[0;97m";
    public static String BLACK_BOLD_BRIGHT = "\\e[1;90m";
    public static String RED_BOLD_BRIGHT = "\\e[1;91m";
    public static String GREEN_BOLD_BRIGHT = "\\e[1;92m";
    public static String YELLOW_BOLD_BRIGHT = "\\e[1;93m";
    public static String BLUE_BOLD_BRIGHT = "\\e[1;94m";
    public static String PURPLE_BOLD_BRIGHT = "\\e[1;95m";
    public static String CYAN_BOLD_BRIGHT = "\\e[1;96m";
    public static String WHITE_BOLD_BRIGHT = "\\e[1;97m";
    public static String BLACK_BACKGROUND_BRIGHT = "\\e[0;100m";
    public static String RED_BACKGROUND_BRIGHT = "\\e[0;101m";
    public static String GREEN_BACKGROUND_BRIGHT = "\\e[0;102m";
    public static String YELLOW_BACKGROUND_BRIGHT = "\\e[0;103m";
    public static String BLUE_BACKGROUND_BRIGHT = "\\e[0;104m";
    public static String PURPLE_BACKGROUND_BRIGHT = "\\e[0;105m";
    public static String CYAN_BACKGROUND_BRIGHT = "\\e[0;106m";
    public static String WHITE_BACKGROUND_BRIGHT = "\\e[0;107m";

    public static String black(String string) {
        return String.format("%s%s%s", BLACK, string, RESET);
    }

    public static String red(String string) {
        return String.format("%s%s%s", RED, string, RESET);
    }

    public static String green(String string) {
        return String.format("%s%s%s", GREEN, string, RESET);
    }

    public static String yellow(String string) {
        return String.format("%s%s%s", YELLOW, string, RESET);
    }

    public static String blue(String string) {
        return String.format("%s%s%s", BLUE, string, RESET);
    }

    public static String purple(String string) {
        return String.format("%s%s%s", PURPLE, string, RESET);
    }

    public static String cyan(String string) {
        return String.format("%s%s%s", CYAN, string, RESET);
    }

    public static String white(String string) {
        return String.format("%s%s%s", WHITE, string, RESET);
    }

    public static String black_bold(String string) {
        return String.format("%s%s%s", BLACK_BOLD, string, RESET);
    }

    public static String red_bold(String string) {
        return String.format("%s%s%s", RED_BOLD, string, RESET);
    }

    public static String green_bold(String string) {
        return String.format("%s%s%s", GREEN_BOLD, string, RESET);
    }

    public static String yellow_bold(String string) {
        return String.format("%s%s%s", YELLOW_BOLD, string, RESET);
    }

    public static String blue_bold(String string) {
        return String.format("%s%s%s", BLUE_BOLD, string, RESET);
    }

    public static String purple_bold(String string) {
        return String.format("%s%s%s", PURPLE_BOLD, string, RESET);
    }

    public static String cyan_bold(String string) {
        return String.format("%s%s%s", CYAN_BOLD, string, RESET);
    }

    public static String white_bold(String string) {
        return String.format("%s%s%s", WHITE_BOLD, string, RESET);
    }

    public static String black_underline(String string) {
        return String.format("%s%s%s", BLACK_UNDERLINE, string, RESET);
    }

    public static String red_underline(String string) {
        return String.format("%s%s%s", RED_UNDERLINE, string, RESET);
    }

    public static String green_underline(String string) {
        return String.format("%s%s%s", GREEN_UNDERLINE, string, RESET);
    }

    public static String yellow_underline(String string) {
        return String.format("%s%s%s", YELLOW_UNDERLINE, string, RESET);
    }

    public static String blue_underline(String string) {
        return String.format("%s%s%s", BLUE_UNDERLINE, string, RESET);
    }

    public static String purple_underline(String string) {
        return String.format("%s%s%s", PURPLE_UNDERLINE, string, RESET);
    }

    public static String cyan_underline(String string) {
        return String.format("%s%s%s", CYAN_UNDERLINE, string, RESET);
    }

    public static String white_underline(String string) {
        return String.format("%s%s%s", WHITE_UNDERLINE, string, RESET);
    }

    public static String black_background(String string) {
        return String.format("%s%s%s", BLACK_BACKGROUND, string, RESET);
    }

    public static String red_background(String string) {
        return String.format("%s%s%s", RED_BACKGROUND, string, RESET);
    }

    public static String green_background(String string) {
        return String.format("%s%s%s", GREEN_BACKGROUND, string, RESET);
    }

    public static String yellow_background(String string) {
        return String.format("%s%s%s", YELLOW_BACKGROUND, string, RESET);
    }

    public static String blue_background(String string) {
        return String.format("%s%s%s", BLUE_BACKGROUND, string, RESET);
    }

    public static String purple_background(String string) {
        return String.format("%s%s%s", PURPLE_BACKGROUND, string, RESET);
    }

    public static String cyan_background(String string) {
        return String.format("%s%s%s", CYAN_BACKGROUND, string, RESET);
    }

    public static String white_background(String string) {
        return String.format("%s%s%s", WHITE_BACKGROUND, string, RESET);
    }

    public static String black_bright(String string) {
        return String.format("%s%s%s", BLACK_BRIGHT, string, RESET);
    }

    public static String red_bright(String string) {
        return String.format("%s%s%s", RED_BRIGHT, string, RESET);
    }

    public static String green_bright(String string) {
        return String.format("%s%s%s", GREEN_BRIGHT, string, RESET);
    }

    public static String yellow_bright(String string) {
        return String.format("%s%s%s", YELLOW_BRIGHT, string, RESET);
    }

    public static String blue_bright(String string) {
        return String.format("%s%s%s", BLUE_BRIGHT, string, RESET);
    }

    public static String purple_bright(String string) {
        return String.format("%s%s%s", PURPLE_BRIGHT, string, RESET);
    }

    public static String cyan_bright(String string) {
        return String.format("%s%s%s", CYAN_BRIGHT, string, RESET);
    }

    public static String white_bright(String string) {
        return String.format("%s%s%s", WHITE_BRIGHT, string, RESET);
    }

    public static String black_bold_bright(String string) {
        return String.format("%s%s%s", BLACK_BOLD_BRIGHT, string, RESET);
    }

    public static String red_bold_bright(String string) {
        return String.format("%s%s%s", RED_BOLD_BRIGHT, string, RESET);
    }

    public static String green_bold_bright(String string) {
        return String.format("%s%s%s", GREEN_BOLD_BRIGHT, string, RESET);
    }

    public static String yellow_bold_bright(String string) {
        return String.format("%s%s%s", YELLOW_BOLD_BRIGHT, string, RESET);
    }

    public static String blue_bold_bright(String string) {
        return String.format("%s%s%s", BLUE_BOLD_BRIGHT, string, RESET);
    }

    public static String purple_bold_bright(String string) {
        return String.format("%s%s%s", PURPLE_BOLD_BRIGHT, string, RESET);
    }

    public static String cyan_bold_bright(String string) {
        return String.format("%s%s%s", CYAN_BOLD_BRIGHT, string, RESET);
    }

    public static String white_bold_bright(String string) {
        return String.format("%s%s%s", WHITE_BOLD_BRIGHT, string, RESET);
    }

    public static String black_background_bright(String string) {
        return String.format("%s%s%s", BLACK_BACKGROUND_BRIGHT, string, RESET);
    }

    public static String red_background_bright(String string) {
        return String.format("%s%s%s", RED_BACKGROUND_BRIGHT, string, RESET);
    }

    public static String green_background_bright(String string) {
        return String.format("%s%s%s", GREEN_BACKGROUND_BRIGHT, string, RESET);
    }

    public static String yellow_background_bright(String string) {
        return String.format("%s%s%s", YELLOW_BACKGROUND_BRIGHT, string, RESET);
    }

    public static String blue_background_bright(String string) {
        return String.format("%s%s%s", BLUE_BACKGROUND_BRIGHT, string, RESET);
    }

    public static String purple_background_bright(String string) {
        return String.format("%s%s%s", PURPLE_BACKGROUND_BRIGHT, string, RESET);
    }

    public static String cyan_background_bright(String string) {
        return String.format("%s%s%s", CYAN_BACKGROUND_BRIGHT, string, RESET);
    }

    public static String white_background_bright(String string) {
        return String.format("%s%s%s", WHITE_BACKGROUND_BRIGHT, string, RESET);
    }
}

File: ansi.py
class Ansi:
    RESET: str = "\\e[0m"
    BLACK: str = "\\e[030m"
    RED: str = "\\e[031m"
    GREEN: str = "\\e[032m"
    YELLOW: str = "\\e[033m"
    BLUE: str = "\\e[034m"
    PURPLE: str = "\\e[035m"
    CYAN: str = "\\e[036m"
    WHITE: str = "\\e[037m"
    BLACK_BOLD: str = "\\e[130m"
    RED_BOLD: str = "\\e[131m"
    GREEN_BOLD: str = "\\e[132m"
    YELLOW_BOLD: str = "\\e[133m"
    BLUE_BOLD: str = "\\e[134m"
    PURPLE_BOLD: str = "\\e[135m"
    CYAN_BOLD: str = "\\e[136m"
    WHITE_BOLD: str = "\\e[137m"
    BLACK_UNDERLINE: str = "\\e[430m"
    RED_UNDERLINE: str = "\\e[431m"
    GREEN_UNDERLINE: str = "\\e[432m"
    YELLOW_UNDERLINE: str = "\\e[433m"
    BLUE_UNDERLINE: str = "\\e[434m"
    PURPLE_UNDERLINE: str = "\\e[435m"
    CYAN_UNDERLINE: str = "\\e[436m"
    WHITE_UNDERLINE: str = "\\e[437m"
    BLACK_BACKGROUND: str = "\\e[40m"
    RED_BACKGROUND: str = "\\e[41m"
    GREEN_BACKGROUND: str = "\\e[42m"
    YELLOW_BACKGROUND: str = "\\e[43m"
    BLUE_BACKGROUND: str = "\\e[44m"
    PURPLE_BACKGROUND: str = "\\e[45m"
    CYAN_BACKGROUND: str = "\\e[46m"
    WHITE_BACKGROUND: str = "\\e[47m"
    BLACK_BRIGHT: str = "\\e[090m"
    RED_BRIGHT: str = "\\e[091m"
    GREEN_BRIGHT: str = "\\e[092m"
    YELLOW_BRIGHT: str = "\\e[093m"
    BLUE_BRIGHT: str = "\\e[094m"
    PURPLE_BRIGHT: str = "\\e[095m"
    CYAN_BRIGHT: str = "\\e[096m"
    WHITE_BRIGHT: str = "\\e[097m"
    BLACK_BOLD_BRIGHT: str = "\\e[190m"
    RED_BOLD_BRIGHT: str = "\\e[191m"
    GREEN_BOLD_BRIGHT: str = "\\e[192m"
    YELLOW_BOLD_BRIGHT: str = "\\e[193m"
    BLUE_BOLD_BRIGHT: str = "\\e[194m"
    PURPLE_BOLD_BRIGHT: str = "\\e[195m"
    CYAN_BOLD_BRIGHT: str = "\\e[196m"
    WHITE_BOLD_BRIGHT: str = "\\e[197m"
    BLACK_BACKGROUND_BRIGHT: str = "\\e[0100m"
    RED_BACKGROUND_BRIGHT: str = "\\e[0101m"
    GREEN_BACKGROUND_BRIGHT: str = "\\e[0102m"
    YELLOW_BACKGROUND_BRIGHT: str = "\\e[0103m"
    BLUE_BACKGROUND_BRIGHT: str = "\\e[0104m"
    PURPLE_BACKGROUND_BRIGHT: str = "\\e[0105m"
    CYAN_BACKGROUND_BRIGHT: str = "\\e[0106m"
    WHITE_BACKGROUND_BRIGHT: str = "\\e[0107m"

    @staticmethod
    def black(string: str) -> str:
        return f'{Ansi.BLACK}{string}{Ansi.RESET}'
    
    @staticmethod
    def red(string: str) -> str:
        return f'{Ansi.RED}{string}{Ansi.RESET}'
    
    @staticmethod
    def green(string: str) -> str:
        return f'{Ansi.GREEN}{string}{Ansi.RESET}'
    
    @staticmethod
    def yellow(string: str) -> str:
        return f'{Ansi.YELLOW}{string}{Ansi.RESET}'
    
    @staticmethod
    def blue(string: str) -> str:
        return f'{Ansi.BLUE}{string}{Ansi.RESET}'
    
    @staticmethod
    def purple(string: str) -> str:
        return f'{Ansi.PURPLE}{string}{Ansi.RESET}'
    
    @staticmethod
    def cyan(string: str) -> str:
        return f'{Ansi.CYAN}{string}{Ansi.RESET}'
    
    @staticmethod
    def white(string: str) -> str:
        return f'{Ansi.WHITE}{string}{Ansi.RESET}'
    
    @staticmethod
    def blackBold(string: str) -> str:
        return f'{Ansi.BLACK_BOLD}{string}{Ansi.RESET}'
    
    @staticmethod
    def redBold(string: str) -> str:
        return f'{Ansi.RED_BOLD}{string}{Ansi.RESET}'
    
    @staticmethod
    def greenBold(string: str) -> str:
        return f'{Ansi.GREEN_BOLD}{string}{Ansi.RESET}'
    
    @staticmethod
    def yellowBold(string: str) -> str:
        return f'{Ansi.YELLOW_BOLD}{string}{Ansi.RESET}'
    
    @staticmethod
    def blueBold(string: str) -> str:
        return f'{Ansi.BLUE_BOLD}{string}{Ansi.RESET}'
    
    @staticmethod
    def purpleBold(string: str) -> str:
        return f'{Ansi.PURPLE_BOLD}{string}{Ansi.RESET}'
    
    @staticmethod
    def cyanBold(string: str) -> str:
        return f'{Ansi.CYAN_BOLD}{string}{Ansi.RESET}'
    
    @staticmethod
    def whiteBold(string: str) -> str:
        return f'{Ansi.WHITE_BOLD}{string}{Ansi.RESET}'
    
    @staticmethod
    def blackUnderline(string: str) -> str:
        return f'{Ansi.BLACK_UNDERLINE}{string}{Ansi.RESET}'
    
    @staticmethod
    def redUnderline(string: str) -> str:
        return f'{Ansi.RED_UNDERLINE}{string}{Ansi.RESET}'
    
    @staticmethod
    def greenUnderline(string: str) -> str:
        return f'{Ansi.GREEN_UNDERLINE}{string}{Ansi.RESET}'
    
    @staticmethod
    def yellowUnderline(string: str) -> str:
        return f'{Ansi.YELLOW_UNDERLINE}{string}{Ansi.RESET}'
    
    @staticmethod
    def blueUnderline(string: str) -> str:
        return f'{Ansi.BLUE_UNDERLINE}{string}{Ansi.RESET}'
    
    @staticmethod
    def purpleUnderline(string: str) -> str:
        return f'{Ansi.PURPLE_UNDERLINE}{string}{Ansi.RESET}'
    
    @staticmethod
    def cyanUnderline(string: str) -> str:
        return f'{Ansi.CYAN_UNDERLINE}{string}{Ansi.RESET}'
    
    @staticmethod
    def whiteUnderline(string: str) -> str:
        return f'{Ansi.WHITE_UNDERLINE}{string}{Ansi.RESET}'
    
    @staticmethod
    def blackBackground(string: str) -> str:
        return f'{Ansi.BLACK_BACKGROUND}{string}{Ansi.RESET}'
    
    @staticmethod
    def redBackground(string: str) -> str:
        return f'{Ansi.RED_BACKGROUND}{string}{Ansi.RESET}'
    
    @staticmethod
    def greenBackground(string: str) -> str:
        return f'{Ansi.GREEN_BACKGROUND}{string}{Ansi.RESET}'
    
    @staticmethod
    def yellowBackground(string: str) -> str:
        return f'{Ansi.YELLOW_BACKGROUND}{string}{Ansi.RESET}'
    
    @staticmethod
    def blueBackground(string: str) -> str:
        return f'{Ansi.BLUE_BACKGROUND}{string}{Ansi.RESET}'
    
    @staticmethod
    def purpleBackground(string: str) -> str:
        return f'{Ansi.PURPLE_BACKGROUND}{string}{Ansi.RESET}'
    
    @staticmethod
    def cyanBackground(string: str) -> str:
        return f'{Ansi.CYAN_BACKGROUND}{string}{Ansi.RESET}'
    
    @staticmethod
    def whiteBackground(string: str) -> str:
        return f'{Ansi.WHITE_BACKGROUND}{string}{Ansi.RESET}'
    
    @staticmethod
    def blackBright(string: str) -> str:
        return f'{Ansi.BLACK_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def redBright(string: str) -> str:
        return f'{Ansi.RED_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def greenBright(string: str) -> str:
        return f'{Ansi.GREEN_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def yellowBright(string: str) -> str:
        return f'{Ansi.YELLOW_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def blueBright(string: str) -> str:
        return f'{Ansi.BLUE_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def purpleBright(string: str) -> str:
        return f'{Ansi.PURPLE_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def cyanBright(string: str) -> str:
        return f'{Ansi.CYAN_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def whiteBright(string: str) -> str:
        return f'{Ansi.WHITE_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def blackBoldBright(string: str) -> str:
        return f'{Ansi.BLACK_BOLD_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def redBoldBright(string: str) -> str:
        return f'{Ansi.RED_BOLD_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def greenBoldBright(string: str) -> str:
        return f'{Ansi.GREEN_BOLD_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def yellowBoldBright(string: str) -> str:
        return f'{Ansi.YELLOW_BOLD_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def blueBoldBright(string: str) -> str:
        return f'{Ansi.BLUE_BOLD_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def purpleBoldBright(string: str) -> str:
        return f'{Ansi.PURPLE_BOLD_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def cyanBoldBright(string: str) -> str:
        return f'{Ansi.CYAN_BOLD_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def whiteBoldBright(string: str) -> str:
        return f'{Ansi.WHITE_BOLD_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def blackBackgroundBright(string: str) -> str:
        return f'{Ansi.BLACK_BACKGROUND_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def redBackgroundBright(string: str) -> str:
        return f'{Ansi.RED_BACKGROUND_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def greenBackgroundBright(string: str) -> str:
        return f'{Ansi.GREEN_BACKGROUND_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def yellowBackgroundBright(string: str) -> str:
        return f'{Ansi.YELLOW_BACKGROUND_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def blueBackgroundBright(string: str) -> str:
        return f'{Ansi.BLUE_BACKGROUND_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def purpleBackgroundBright(string: str) -> str:
        return f'{Ansi.PURPLE_BACKGROUND_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def cyanBackgroundBright(string: str) -> str:
        return f'{Ansi.CYAN_BACKGROUND_BRIGHT}{string}{Ansi.RESET}'
    
    @staticmethod
    def whiteBackgroundBright(string: str) -> str:
        return f'{Ansi.WHITE_BACKGROUND_BRIGHT}{string}{Ansi.RESET}'
    